utils/s2n_init.c

```diff
@@ -27,6 +27,8 @@
 
 #include "openssl/opensslv.h"
 
```

Comment:
Should this be volatile?
Alternatively (and safer) maybe pthread_once() may be even better?

Reply:
My actual use case is not concerned with concurrent calls to s2n_init(). I have one library's code path calling s2n_init() at process start and another lazily calling s2n_init() some time after start.

But I will look into pthread_once() to see if I can cover easily cover the concurrent use case.

Reply:
+1 for pthread_once